spark/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -242,6 +242,9 @@ class DeltaTableHadoopOptionsSuite extends QueryTest
     "addFeatureSupport",
     "as",
     "alias",
+    "clone",
+    "cloneAtTimestamp",
```

Comment:
```suggestion
    "cloneAtVersion",
    "cloneAtTimestamp",
```

Reply:
These are alphabetical, not the order they appear in the DeltaTable class

Reply:
@Kimahriman I see, make sense